src/lib/services/access-service.ts

```diff
@@ -108,6 +108,15 @@ export interface AccessWithRoles {
 
 const isProjectPermission = (permission) => PROJECT_ADMIN.includes(permission);
 
+export const cleanPermissions = (permissions: PermissionRef[] | undefined) => {
```

Comment:
Ensure all incoming permissions use null instead of empty string in the service layer